scipy/stats/_qmc.py

```diff
@@ -727,6 +737,81 @@ def random(self, n: IntNumber = 1) -> np.ndarray:
         """
         # self.num_generated += n
 
+    def integers(
+        self,
+        l_bounds: npt.ArrayLike,
+        *,
+        u_bounds: Optional[npt.ArrayLike] = None,
+        n: IntNumber = 1,
+        endpoint: bool = False,
+        workers: IntNumber = 1
+    ) -> np.ndarray:
+        """
+        Draw `n` integers from `l_bounds` (inclusive) to `u_bounds`
+        (exclusive), or if endpoint=True, `l_bounds` (inclusive) to
+        `u_bounds` (inclusive).
+
+        Parameters
+        ----------
+        l_bounds : int or array-like of ints
```

Comment:
If we're going to match the NumPy interface, might as well name them `low`/`high`?

Reply:
I did not do it, for now, to match the parameters in `scale`.

Reply:
Oops. Well, we can leave it, or if you prefer, there's a decorator for changing it : )

Reply:
I would say for this PR we don't do it and we can definitely think about that for a follow up.